[2024-11-29T05:05:23.657+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-11-29T05:05:23.670+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: data_pipeline_etl.collect_data manual__2024-11-29T05:05:22.653111+00:00 [queued]>
[2024-11-29T05:05:23.676+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: data_pipeline_etl.collect_data manual__2024-11-29T05:05:22.653111+00:00 [queued]>
[2024-11-29T05:05:23.676+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 2
[2024-11-29T05:05:23.685+0000] {taskinstance.py:2889} INFO - Executing <Task(BashOperator): collect_data> on 2024-11-29 05:05:22.653111+00:00
[2024-11-29T05:05:23.691+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=1343) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-11-29T05:05:23.692+0000] {standard_task_runner.py:72} INFO - Started process 1344 to run task
[2024-11-29T05:05:23.691+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'data_pipeline_etl', 'collect_data', 'manual__2024-11-29T05:05:22.653111+00:00', '--job-id', '57', '--raw', '--subdir', 'DAGS_FOLDER/etl_pipeline_auto.py', '--cfg-path', '/tmp/tmpihgrs8_x']
[2024-11-29T05:05:23.693+0000] {standard_task_runner.py:105} INFO - Job 57: Subtask collect_data
[2024-11-29T05:05:23.727+0000] {task_command.py:467} INFO - Running <TaskInstance: data_pipeline_etl.collect_data manual__2024-11-29T05:05:22.653111+00:00 [running]> on host f8512d2e4f1d
[2024-11-29T05:05:23.785+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='data_pipeline_etl' AIRFLOW_CTX_TASK_ID='collect_data' AIRFLOW_CTX_EXECUTION_DATE='2024-11-29T05:05:22.653111+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-11-29T05:05:22.653111+00:00'
[2024-11-29T05:05:23.785+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-11-29T05:05:23.821+0000] {subprocess.py:78} INFO - Tmp dir root location: /tmp
[2024-11-29T05:05:23.822+0000] {subprocess.py:88} INFO - Running command: ['/usr/bin/bash', '-c', 'python /opt/***/data-pipeline-project/product_data_scraper.py']
[2024-11-29T05:05:23.829+0000] {subprocess.py:99} INFO - Output:
[2024-11-29T05:06:01.462+0000] {subprocess.py:106} INFO - 320번 데이터 수집 시작
[2024-11-29T05:06:01.466+0000] {subprocess.py:106} INFO - 페이지 2 데이터 수집 시작 (카테고리: 320)
[2024-11-29T05:06:01.467+0000] {subprocess.py:106} INFO - 데이터 수집중2
[2024-11-29T05:06:01.470+0000] {subprocess.py:106} INFO - 페이지 2 데이터 수집 완료 (카테고리: 320)
[2024-11-29T05:06:01.470+0000] {subprocess.py:106} INFO - 페이지 1 데이터 수집 시작 (카테고리: 320)
[2024-11-29T05:06:01.470+0000] {subprocess.py:106} INFO - 데이터 수집중1
[2024-11-29T05:06:01.470+0000] {subprocess.py:106} INFO - 페이지 1 데이터 수집 완료 (카테고리: 320)
[2024-11-29T05:06:01.470+0000] {subprocess.py:106} INFO - {'/opt/***/collectedData/mans_category/mans_category_20241129_050544.csv'}
[2024-11-29T05:06:01.471+0000] {subprocess.py:106} INFO - 데이터 수집 및 저장 완료
[2024-11-29T05:06:01.471+0000] {subprocess.py:106} INFO - 310번 데이터 수집 시작
[2024-11-29T05:06:01.471+0000] {subprocess.py:106} INFO - 페이지 2 데이터 수집 시작 (카테고리: 310)
[2024-11-29T05:06:01.471+0000] {subprocess.py:106} INFO - 데이터 수집중2
[2024-11-29T05:06:01.471+0000] {subprocess.py:106} INFO - 페이지 2 데이터 수집 완료 (카테고리: 310)
[2024-11-29T05:06:01.471+0000] {subprocess.py:106} INFO - 페이지 1 데이터 수집 시작 (카테고리: 310)
[2024-11-29T05:06:01.471+0000] {subprocess.py:106} INFO - 데이터 수집중1
[2024-11-29T05:06:01.471+0000] {subprocess.py:106} INFO - 페이지 1 데이터 수집 완료 (카테고리: 310)
[2024-11-29T05:06:01.471+0000] {subprocess.py:106} INFO - {'/opt/***/collectedData/woman_category/woman_category_20241129_050601.csv'}
[2024-11-29T05:06:01.472+0000] {subprocess.py:106} INFO - 데이터 수집 및 저장 완료
[2024-11-29T05:06:01.527+0000] {subprocess.py:110} INFO - Command exited with return code 0
[2024-11-29T05:06:01.544+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-11-29T05:06:01.545+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=data_pipeline_etl, task_id=collect_data, run_id=manual__2024-11-29T05:05:22.653111+00:00, execution_date=20241129T050522, start_date=20241129T050523, end_date=20241129T050601
[2024-11-29T05:06:01.557+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-11-29T05:06:01.571+0000] {taskinstance.py:3895} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-11-29T05:06:01.572+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
